server/views/describe-the-item.html

```diff
@@ -40,7 +40,7 @@
             html: '<span class="govuk-!-font-weight-bold">Where is the ivory on it?</span>'
           },
           hint: {
-            text: "For example, 'scabbard has ivory inlay', 'chest has ivory knobs'"
+            html: "For example, 'scabbard has ivory inlay', 'chest has ivory knobs'<br>No more than 128 characters, so that it will fit on your certificate."
```

Comment:
This additional hint text should be on the `distinguishingFeatures` field, not this `whereIsIvory` field and it should say 243 rather than 128

Reply:
Yes, my bad :) Corrected it.